sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuListUtils.scala

```diff
@@ -30,9 +30,8 @@ object GpuListUtils extends Arm {
    * both listCol and newDataCol alive longer than the returned ColumnView.
    * @param listCol the list column to use as a template
    * @param newDataCol the new data column.
-   * @return a new ColumnView.
-   * @throws scala.IllegalArgumentException if data column does not match the original data column
-   *                                        in size.
+   * @return a new ColumnView. But throws an scala.IllegalArgumentException if data column does
```

Comment:
The only reason `scala.IllegalArgumentException` was originally used was to try to appease scaladoc.  We should just use IllegalArgumentException.
```suggestion
   * @return a new ColumnView. Throws IllegalArgumentException if data column does
```

Reply:
Updated